<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="7" total="7" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Smoke" duration-ms="4959" started-at="2018-08-29T11:21:40Z" finished-at="2018-08-29T11:21:45Z">
    <groups>
    </groups>
    <test name="SanitySuite2018" duration-ms="86" started-at="2018-08-29T11:21:40Z" finished-at="2018-08-29T11:21:41Z">
      <class name="com.hasbro.sanity.SanitySuite2018">
        <test-method status="SKIP" signature="checkTestcaseSkip()[pri:0, instance:com.hasbro.sanity.SanitySuite2018@69ea3742]" name="checkTestcaseSkip" is-config="true" duration-ms="0" started-at="2018-08-29T16:51:40Z" finished-at="2018-08-29T16:51:40Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping SanitySuite2018 as runmode is set to no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping SanitySuite2018 as runmode is set to no
	at com.hasbro.sanity.SanitySuite2018.checkTestcaseSkip(SanitySuite2018.java:400)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTestcaseSkip -->
        <test-method status="FAIL" signature="tcSCFunctional([Ljava.lang.String;)[pri:0, instance:com.hasbro.sanity.SanitySuite2018@69ea3742]" name="tcSCFunctional" duration-ms="0" started-at="2018-08-29T16:51:41Z" data-provider="testDataTest" finished-at="2018-08-29T16:51:41Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuite2018.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuite2018.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:426)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tcSCFunctional -->
        <test-method status="SKIP" signature="reportTestcaseResult()[pri:0, instance:com.hasbro.sanity.SanitySuite2018@69ea3742]" name="reportTestcaseResult" is-config="true" duration-ms="0" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportTestcaseResult -->
      </class> <!-- com.hasbro.sanity.SanitySuite2018 -->
    </test> <!-- SanitySuite2018 -->
    <test name="SanitySuite1" duration-ms="30" started-at="2018-08-29T11:21:41Z" finished-at="2018-08-29T11:21:41Z">
      <class name="com.hasbro.sanity.SanitySuite1">
        <test-method status="SKIP" signature="checkTestcaseSkip()[pri:0, instance:com.hasbro.sanity.SanitySuite1@6d7b4f4c]" name="checkTestcaseSkip" is-config="true" duration-ms="0" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:41Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping SanitySuite1 as runmode is set to no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping SanitySuite1 as runmode is set to no
	at com.hasbro.sanity.SanitySuite1.checkTestcaseSkip(SanitySuite1.java:17967)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTestcaseSkip -->
        <test-method status="FAIL" signature="tcSCFunctional([Ljava.lang.String;)[pri:0, instance:com.hasbro.sanity.SanitySuite1@6d7b4f4c]" name="tcSCFunctional" duration-ms="0" started-at="2018-08-29T16:51:41Z" data-provider="testDataTest" finished-at="2018-08-29T16:51:41Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuite1.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuite1.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:426)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tcSCFunctional -->
        <test-method status="SKIP" signature="reportTestcaseResult()[pri:0, instance:com.hasbro.sanity.SanitySuite1@6d7b4f4c]" name="reportTestcaseResult" is-config="true" duration-ms="0" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportTestcaseResult -->
      </class> <!-- com.hasbro.sanity.SanitySuite1 -->
    </test> <!-- SanitySuite1 -->
    <test name="SanitySuite3" duration-ms="2" started-at="2018-08-29T11:21:41Z" finished-at="2018-08-29T11:21:41Z">
      <class name="com.hasbro.sanity.SanitySuite3">
        <test-method status="SKIP" signature="checkTestcaseSkip()[pri:0, instance:com.hasbro.sanity.SanitySuite3@6b09bb57]" name="checkTestcaseSkip" is-config="true" duration-ms="0" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:41Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping SanitySuite3 as runmode is set to no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping SanitySuite3 as runmode is set to no
	at com.hasbro.sanity.SanitySuite3.checkTestcaseSkip(SanitySuite3.java:3787)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTestcaseSkip -->
        <test-method status="FAIL" signature="tcSCFunctional([Ljava.lang.String;)[pri:0, instance:com.hasbro.sanity.SanitySuite3@6b09bb57]" name="tcSCFunctional" duration-ms="0" started-at="2018-08-29T16:51:41Z" data-provider="testDataTest" finished-at="2018-08-29T16:51:41Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuite3.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuite3.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:426)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tcSCFunctional -->
        <test-method status="SKIP" signature="reportTestcaseResult()[pri:0, instance:com.hasbro.sanity.SanitySuite3@6b09bb57]" name="reportTestcaseResult" is-config="true" duration-ms="0" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportTestcaseResult -->
      </class> <!-- com.hasbro.sanity.SanitySuite3 -->
    </test> <!-- SanitySuite3 -->
    <test name="SanitySuite2" duration-ms="3" started-at="2018-08-29T11:21:41Z" finished-at="2018-08-29T11:21:41Z">
      <class name="com.hasbro.sanity.SanitySuite2">
        <test-method status="SKIP" signature="checkTestcaseSkip()[pri:0, instance:com.hasbro.sanity.SanitySuite2@185d8b6]" name="checkTestcaseSkip" is-config="true" duration-ms="0" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:41Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping SanitySuite2 as runmode is set to no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping SanitySuite2 as runmode is set to no
	at com.hasbro.sanity.SanitySuite2.checkTestcaseSkip(SanitySuite2.java:7067)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTestcaseSkip -->
        <test-method status="FAIL" signature="tcSCFunctional([Ljava.lang.String;)[pri:0, instance:com.hasbro.sanity.SanitySuite2@185d8b6]" name="tcSCFunctional" duration-ms="0" started-at="2018-08-29T16:51:41Z" data-provider="testDataTest" finished-at="2018-08-29T16:51:41Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuite2.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuite2.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:426)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tcSCFunctional -->
        <test-method status="SKIP" signature="reportTestcaseResult()[pri:0, instance:com.hasbro.sanity.SanitySuite2@185d8b6]" name="reportTestcaseResult" is-config="true" duration-ms="0" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportTestcaseResult -->
      </class> <!-- com.hasbro.sanity.SanitySuite2 -->
    </test> <!-- SanitySuite2 -->
    <test name="SanitySuite_AssortmentRetail" duration-ms="9" started-at="2018-08-29T11:21:41Z" finished-at="2018-08-29T11:21:41Z">
      <class name="com.hasbro.sanity.SanitySuite_AssortmentRetail">
        <test-method status="SKIP" signature="checkTestcaseSkip()[pri:0, instance:com.hasbro.sanity.SanitySuite_AssortmentRetail@57536d79]" name="checkTestcaseSkip" is-config="true" duration-ms="0" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:41Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping SanitySuite_AssortmentRetail as runmode is set to no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping SanitySuite_AssortmentRetail as runmode is set to no
	at com.hasbro.sanity.SanitySuite_AssortmentRetail.checkTestcaseSkip(SanitySuite_AssortmentRetail.java:2063)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTestcaseSkip -->
        <test-method status="FAIL" signature="tcSCFunctional([Ljava.lang.String;)[pri:0, instance:com.hasbro.sanity.SanitySuite_AssortmentRetail@57536d79]" name="tcSCFunctional" duration-ms="0" started-at="2018-08-29T16:51:41Z" data-provider="testDataTest" finished-at="2018-08-29T16:51:41Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuite_AssortmentRetail.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuite_AssortmentRetail.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:426)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tcSCFunctional -->
        <test-method status="SKIP" signature="reportTestcaseResult()[pri:0, instance:com.hasbro.sanity.SanitySuite_AssortmentRetail@57536d79]" name="reportTestcaseResult" is-config="true" duration-ms="0" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportTestcaseResult -->
      </class> <!-- com.hasbro.sanity.SanitySuite_AssortmentRetail -->
    </test> <!-- SanitySuite_AssortmentRetail -->
    <test name="SanitySuiteUpdateProductName" duration-ms="2" started-at="2018-08-29T11:21:41Z" finished-at="2018-08-29T11:21:41Z">
      <class name="com.hasbro.sanity.SanitySuiteUpdateProductName">
        <test-method status="SKIP" signature="checkTestcaseSkip()[pri:0, instance:com.hasbro.sanity.SanitySuiteUpdateProductName@7d907bac]" name="checkTestcaseSkip" is-config="true" duration-ms="0" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:41Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping SanitySuiteUpdateProductName as runmode is set to no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping SanitySuiteUpdateProductName as runmode is set to no
	at com.hasbro.sanity.SanitySuiteUpdateProductName.checkTestcaseSkip(SanitySuiteUpdateProductName.java:1371)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTestcaseSkip -->
        <test-method status="FAIL" signature="tcSCFunctional([Ljava.lang.String;)[pri:0, instance:com.hasbro.sanity.SanitySuiteUpdateProductName@7d907bac]" name="tcSCFunctional" duration-ms="0" started-at="2018-08-29T16:51:41Z" data-provider="testDataTest" finished-at="2018-08-29T16:51:41Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuiteUpdateProductName.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuiteUpdateProductName.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:426)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tcSCFunctional -->
        <test-method status="SKIP" signature="reportTestcaseResult()[pri:0, instance:com.hasbro.sanity.SanitySuiteUpdateProductName@7d907bac]" name="reportTestcaseResult" is-config="true" duration-ms="0" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportTestcaseResult -->
      </class> <!-- com.hasbro.sanity.SanitySuiteUpdateProductName -->
    </test> <!-- SanitySuiteUpdateProductName -->
    <test name="SanitySuite5_WorkFlow" duration-ms="4619" started-at="2018-08-29T11:21:41Z" finished-at="2018-08-29T11:21:45Z">
      <class name="com.hasbro.sanity.SanitySuite5_WorkFlow">
        <test-method status="PASS" signature="checkSuiteSkipTest()[pri:0, instance:com.hasbro.sanity.SanitySuite5_WorkFlow@65e2dbf3]" name="checkSuiteSkipTest" is-config="true" duration-ms="7323" started-at="2018-08-29T16:51:33Z" finished-at="2018-08-29T16:51:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkSuiteSkipTest -->
        <test-method status="PASS" signature="checkTestcaseSkip()[pri:0, instance:com.hasbro.sanity.SanitySuite5_WorkFlow@65e2dbf3]" name="checkTestcaseSkip" is-config="true" duration-ms="9" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTestcaseSkip -->
        <test-method status="PASS" signature="reportTestcaseResult()[pri:0, instance:com.hasbro.sanity.SanitySuite5_WorkFlow@65e2dbf3]" name="reportTestcaseResult" is-config="true" duration-ms="4609" started-at="2018-08-29T16:51:41Z" finished-at="2018-08-29T16:51:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportTestcaseResult -->
        <test-method status="FAIL" signature="tcSCFunctional([Ljava.lang.String;)[pri:0, instance:com.hasbro.sanity.SanitySuite5_WorkFlow@65e2dbf3]" name="tcSCFunctional" duration-ms="0" started-at="2018-08-29T16:51:41Z" data-provider="testDataTest" finished-at="2018-08-29T16:51:41Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuite5_WorkFlow.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[] com.hasbro.sanity.SanitySuite5_WorkFlow.testDataTest() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:426)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tcSCFunctional -->
      </class> <!-- com.hasbro.sanity.SanitySuite5_WorkFlow -->
    </test> <!-- SanitySuite5_WorkFlow -->
  </suite> <!-- Smoke -->
</testng-results>
